src/operations/query/convertGraphQLParameters.js

```diff
@@ -24,7 +25,9 @@ function convertGraphQLParameters (queryParameterValue, args, queryParameter) {
                         queryParameterValue = [];
                     } else {
                         // handle SearchString
-                        queryParameterValue = queryParameterValue.value || queryParameterValue.values;
+                        if (queryParameterValue.value || queryParameterValue.values) {
```

Comment:
Was not able to replicated error in that part of the code.
There was an error in query

We're not handling 'value_string' here either, because it is part of the multiple values problem
generated query
db.Observation_4_0_0.find({'$and':[{'$or':[{'valueQuantity':{'$exists':true}},{'valueCodeableConcept':{'$exists':true}},{'valueString':{'$exists':true}},{'valueBoolean':{'$exists':true}},{'valueInteger':{'$exists':true}},{'valueRange':{'$exists':true}},{'valueRatio':{'$exists':true}},{'valueSampledData':{'$exists':true}},{'valueTime':{'$exists':true}},{'valueDateTime':{'$exists':true}},{'valuePeriod':{'$exists':true}}]}